hello-world-apk/build.gradle

```diff
@@ -35,3 +35,10 @@ android {
         }
     }
 }
+
+dependencies {
+    //included just to test wildcard pom generation
+    compile group: 'org.apache.commons', name: 'commons-io', version: '1.3.+'
+    compile group: 'org.apache.commons', name: 'commons-io', version: '1.3.+'
```

Comment:
oh it wasn't supposed to be a dup. i was just picking libraries are random,
should have been commons-io

On Tue, Aug 23, 2016 at 10:25 PM, Chris Doyle <notifications@github.com>
wrote:

> In hello-world-apk/build.gradle
> <https://github.com/chrisdoyle/gradle-fury/pull/28#discussion_r75985573>:
>
> > @@ -35,3 +35,10 @@ android {
> >          }
> >      }
> >  }
> > +
> > +dependencies {
> > +    //included just to test wildcard pom generation
> > +    compile group: 'org.apache.commons', name: 'commons-io', version: '1.3.+'
> > +    compile group: 'org.apache.commons', name: 'commons-io', version: '1.3.+'
>
> We should also test for a wildcard like 3.+, maybe replace one of these
> duplicate commons-io deps with everyone's favorite commons-lang3 to test
> that out?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/chrisdoyle/gradle-fury/pull/28/files/e10b901c05de729772072af468c2ebd03154fa44#r75985573>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB4kGSxqE194H-nDp0jl9emkDFjX30Etks5qi6t9gaJpZM4JrY8E>
> .
>